encryptor/config/encryptionSettings.go

```diff
@@ -243,9 +254,15 @@ func (s *BasicColumnEncryptionSetting) Init() (err error) {
 	if s.ReEncryptToAcraBlock != nil && *s.ReEncryptToAcraBlock {
 		s.settingMask |= SettingReEncryptionFlag
 	}
+
+	tokenizeEnabled := (s.Tokenized != nil && *s.Tokenized)
+	if s.Tokenized != nil && !*s.Tokenized && s.TokenType != "" {
```

Comment:
let's log warning that `Tokenized` option is deprecated if it is `!= nil`

Reply:
I suggest smth like "Setting `tokenized` flag is not necessary anymore and will be ignored".

Reply:
Thought about this, but it would require passing `logger`, which either require changing function interface to pass it, or use `log` lib directly. Which is better?

Reply:
at this case we can use `log` lib directly because it called at the start of program in global space, not in the specific user's session or something like that